packages/office-addin-debugging/src/start.ts

```diff
@@ -191,13 +192,13 @@ export async function startDebugging(manifestPath: string, appType: AppType, app
     sourceBundleUrlComponents?: devSettings.SourceBundleUrlComponents,
     devServerCommandLine?: string, devServerPort?: number,
     packagerCommandLine?: string, packagerHost?: string, packagerPort?: string,
-    enableDebugging: boolean = true, enableLiveReload: boolean = true, openDevTools: boolean = false) {
+    enableDebugging: boolean = true, enableLiveReload: boolean = true,
+    openDevTools: boolean = false, document?: string) {
 
     try {
 
         const isWindowsPlatform = (process.platform === "win32");
         const isDesktopAppType = (appType === AppType.Desktop);
-        const isWebAppType = (appType === AppType.Web);
```

Comment:
Why is this removed? We may not want to assume that if not desktop that it implies web as we could add another platform type in the future.

Reply:
I think it is because there is now not a specific check and the sideload is done for all appTypes, which is good.